form/3d/p5js/Lines for Space/index.md

```diff
@@ -0,0 +1,9 @@
+---  
+layout: p5js
+author: Jeanette Knipp
+author-url: https://github.com/Lellahahaha
+title: Lines for Space
+tags: [P5.js, Form, Law of Space]
+permalink: form/Lines for Space
```

Comment:
This permalink and the folder name need to be URL save. Please change it to

    /form/lines-for-space/